CVs/input4MIPs_source_id.json

```diff
@@ -129,5 +129,13 @@
         "license_id":"CC BY 4.0",
         "mip_era":"CMIP6Plus",
         "source_version":"1.1.3"
+    },
+    "UofMD-landState-3-0":{
+        "contact":"lchini@umd.edu",
+        "further_info_url":"www.tbd.invalid",
+        "institution_id":"UofMD",
+        "license_id":"CC BY 4.0",
+        "mip_era":"CMIP6Plus",
+        "source_version":"3.0"
     }
```

Comment:
@lchini this is the info to double check

Reply:
Yes, this looks good. I think we should add George Hurtt to the contact field (gchurtt@umd.edu). Also, the further info url should be: https://luh.umd.edu

Reply:
Thanks, updated

Reply:
Thanks, updated